Taarafo.Core/Brokers/Storages/StorageBroker.GroupMemberships.References.cs

```diff
@@ -0,0 +1,25 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using Microsoft.EntityFrameworkCore;
+using Taarafo.Core.Models.GroupMemberships;
+using Taarafo.Core.Models.Profiles;
+
+namespace Taarafo.Core.Brokers.Storages
+{
+	public partial class StorageBroker
+	{
+		private static void AddGroupMembershipReferences(ModelBuilder modelBuilder)
+		{
+			modelBuilder.Entity<GroupMembership>()
+				.HasMany(groupMembership => groupMembership.GroupMemberships)
```

Comment:
This should be `.HasOne` because you can only have one Profile.

Which means that you will need to go to GroupMembership and add a `Profile` property in there.

So your code would look like this: `HasOne(groupMembership => groupMembership.Profile)`